client/src/components/Workflow/TrsServerSelection.vue

```diff
@@ -0,0 +1,68 @@
+<!-- Modelled after Sam's ToolShed ServerSelection.vue -->
+<template>
+    <span v-if="!loading" class="m-1 text-muted">
+        <span v-if="showDropdown" class="dropdown">
+            <b-link
+                id="dropdownTrsServer"
+                data-toggle="dropdown"
+                aria-haspopup="true"
+                aria-expanded="false"
+                class="font-weight-bold"
+            >
+                {{ selection.label }}
+                <span class="fa fa-caret-down" />
+            </b-link>
+            <div class="dropdown-menu" aria-labelledby="dropdownTrsServer">
+                <a
+                    v-for="selection in trsServers"
+                    :key="selection.id"
+                    class="dropdown-item"
+                    href="javascript:void(0)"
```

Comment:
I get that this was needed in old days, but is this still needed in Vue days?

Reply:
I don't know for sure - I got it by copy and pasting code that was last modified with this commit (https://github.com/galaxyproject/galaxy/pull/8334) which seems fairly recent.